web-console/developer/pipelines/pipelines.md

```diff
@@ -1,7 +1,191 @@
\ No newline at end of file
+## All Projects in project selector
+When a project is not in context, clicking Pipelines will prompt the user to select from a list of projects:
+![Pipelines Resource Table - All Projects](img/PL-ListView-AllProjects.png)
+
+## Pipelines Resource Table
+When a project is in context, clicking “Pipelines” will display the Pipelines Resources in that project.
+![Pipelines Resource Table](img/PL-ListView.png)
+
+- Navigation item: Pipelines  
+- Secondary masthead is visible and includes the project selector 
+- Page title is Pipelines
+- Filtering
+    - OpenShift filter toggle component is present, filtering on the Last Run Status
+    - OpenShift filter by name component is present
+- Sorting
+    - All columns are sortable.  
+    
+    Columns include:
+    1. Name
+    Resource badge + pipeline name are presented as a link.  Clicking on the link navigates to the Pipeline Details View
+    1. Last Pipeline Run
+    Resource badge + name of last pipeline run.  Clicking on the link navigates to the Pipeline Run Details View
+    1. Last Run Status
+    1. Last Status Check
+    1. Last Run Started
+
+## Pipelines Resource Details
+### Pipelines - Overview tab
+![Pipelines Overview tab](img/PL-Details-Overview.png)
+
+### Pipelines - YAML tab
+![Pipelines YAML tab](img/PL-Details-YAML.png)
+
+### Pipelines - Pipeline Runs tab
+![Pipelines PLR tab](img/PL-Details-PLR.png)
+
+### Pipelines - Parameters tab
+![Pipelines Parameters tab](img/PL-Details-Parameters.png)
+
+### Pipelines - Resources tab
+![Pipelines Resources tab](img/PL-Details-Resources.png)
+
+There are 2 columns in the Pipelines resource tab:
+1. Name - resource name
+1. Resource Type - a dropdown including git, image, cluster & storage
+
+#### Adding a Resource
+The general behavior of adding a resource is consistent with the Environments tab of many workloads pages.  
+
+When the user clicks the Add Resource link, a new row is added.  Hints are shown in the input fields and an info inline notification is displayed. 
+![Add a resource-01](img/PL-Details-Resources-AddResource01.png)
+
+The resource type can be one of the following: git, cluster, image, storage
+![Add a resource-02](img/PL-Details-Resources-AddResource02.png)
+
+When the user clicks Save, a success inline notification is displayed. 
+![Add a resource-04](img/PL-Details-Resources-AddResource04.png)
+
+#### Removing a Resource
+When the user hovers over the Remove Resource icon, a Remove Resource tooltip should be displayed
+![Remove a resource-00](img/PL-Details-Resources-RemoveResource.png)
+
+When the user clicks the Remove Resource icon
+The row is removed
+An info inline notification is shown 
+![Remove a resource-01](img/PL-Details-Resources-RemoveResource01.png)
+
+When the user clicks Save
+An success inline notification is shown 
+![Remove a resource-02](img/PL-Details-Resources-RemoveResource02.png)
+
+
+## Pipeline Actions
+Supported pipeline actions are: 
+1. Restart last run
```

Comment:
What is the difference of "restart last run" vs "start"?

Reply:
@rileyhuston restarting last run will restart the last pipeline run ( with the same config info ).  Starting a pipeline will create a new pipeline based on the information captured in the dialog.